elements/pf-back-to-top/test/pf-back-to-top.spec.ts

```diff
@@ -56,6 +56,10 @@ describe('<pf-back-to-top>', function() {
       expect(children).to.be.undefined;
     });
 
+    it('should not be accessible', function() {
+      expect(element).to.not.be.accessible;
```

Comment:
This won't do anything. You need to call and await accessible
```suggestion
      await expect(element).to.not.be.accessible();
```

I think snapshot would be better suited for this assertion